src/security-council-mgmt/SecurityCouncilManager.sol

```diff
@@ -122,6 +150,33 @@ contract SecurityCouncilManager is
         for (uint256 i = 0; i < _securityCouncils.length; i++) {
             _addSecurityCouncil(_securityCouncils[i]);
         }
+
+        setMinRotationPeriodImpl(_minRotationPeriod);
+
+        __EIP712_init_unchained("SecurityCouncilManager", "1");
+    }
+
+    function postUpgradeInit(uint256 _minRotationPeriod, address minRotationPeriodSetter)
```

Comment:
__EIP712_init_unchained wasn't initialized in postUpgradeInit

Reply:
good catch, now set name and version hash instead